### PR TITLE
[BULK] DocuTune - Updates to Azure AD rebranding guidance and DocuTune configuration (part 5)

### PR DESCRIPTION
**Pull request by alexbuckgit**
Applying Azure AD rebranding changes as announced on July 11, 2023 and outlined in https://aka.ms/AzureADNewName. Changes are part of the Microsoft-wide rebranding effort.

CorrelationId: 2520d58e-25ff-469e-af14-2834d2a1505e
PointOfContact: @CelesteDG

#docutune

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Review by mikefrobbins — DISMISSED** [review]
LGTM

**Comment by CelesteDG** [comment]
@mikefrobbins @alexbuckgit Has this build stalled? Do we need to close and reopen the PR?

**Comment by mikefrobbins** [comment]
@VeryEarly Please merge this PR unless there is any feedback or questions. - Thank you

File: src/Sql/Sql/help/New-AzSqlInstance.md [hunk 1]
```diff
@@ -238,7 +238,9 @@ MaintenanceConfigurationId			: /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxx
 
 This command creates a new instance with maintenance configuration MI_2
 
-### Example 7: Create a new instance with External(Azure Active Directory) Administrator, Azure Active Directory Only Authentication and no SqlAdministratorCredentials
+<a name='example-7-create-a-new-instance-with-externalazure-active-directory-administrator-azure-active-directory-only-authentication-and-no-sqladministratorcredentials'></a>
+
```
Comment (BethanyZhou) [inline src/Sql/Sql/help/New-AzSqlInstance.md:242]:
```suggestion
```

File: src/Sql/Sql/help/New-AzSqlServer.md [hunk 2]
```diff
@@ -46,7 +46,9 @@ Tags                     :
 
 This command creates a version 12 Azure SQL Database server.
 
-### Example 2: Create a new Azure SQL Database server with External(Azure Active Directory) Administrator, Azure Active Directory Only Authentication and no SqlAdministratorCredentials
+<a name='example-2-create-a-new-azure-sql-database-server-with-externalazure-active-directory-administrator-azure-active-directory-only-authentication-and-no-sqladministratorcredentials'></a>
+
```
Comment (BethanyZhou) [inline src/Sql/Sql/help/New-AzSqlServer.md:50]:
```suggestion
```

**Review by mikefrobbins — APPROVED** [review]
LGTM

**Pull request merged**